_data/versions.yml

```diff
@@ -1,3 +1,8 @@
+- name: "4.3"
+  flavor: "rc1"
+  release_date: "8 December 2023"
+  release_notes: "/article/release-candidate-godot-4-2-1-rc-1/"
```

Comment:
@YuriSizov Is it correct not to have a `releases` section when there's only one release?

Reply:
Of course it's better with `name: "4.2.1"` :P

Reply:
Nothing should break if it's amiss, so yeah